src/controller.cpp

```diff
@@ -360,8 +360,8 @@ int Controller::run(const CliArgsParser& args)
 	}
 	const auto error_message = urlcfg->reload();
 	if (error_message.has_value()) {
-		std::cout << error_message.value() << std::endl << std::endl;
-		return EXIT_FAILURE;
+		std::cout << error_message.value().message << std::endl << std::endl;
+		//return EXIT_FAILURE;
```

Comment:
Don't comment out code, just remove it ;) We have Git, so we can restore anything at any time.